src/sync/contracts.js

```diff
@@ -1,34 +1,32 @@
+    FinalResultSet: '14959b24f45a8f41b814b331ae09533db9d7e7962ca200e484f849a1fd1955aa',
```

Comment:
signature should be dynamically calculated

Reply:
can change later when you decide if we should hardcode the eventName here or inside call.